src/materialized/src/lib.rs

```diff
@@ -13,15 +13,20 @@
 //! [differential dataflow]: ../differential_dataflow/index.html
 //! [timely dataflow]: ../timely/index.html
 
-use std::convert::TryInto;
 use std::env;
 use std::net::SocketAddr;
 use std::path::PathBuf;
-use std::time::Duration;
+use std::time::{Duration, Instant};
+use std::{convert::TryInto, sync::Arc};
 
 use compile_time_run::run_command_str;
 use futures::StreamExt;
 use openssl::ssl::{SslAcceptor, SslFiletype, SslMethod, SslVerifyMode};
+use ore::{
```

Comment:
If you could configure your rust-analyzer to use importStyle "last" instead of this, when convenient, that'd be swell!